Makefile

```diff
@@ -5,6 +5,18 @@ lib: node_modules
 	$(BIN)/babel --out-dir $@ src --source-maps inline
 	$(BIN)/postcss --use autoprefixer --dir $@ src/index.css
 
+build:
+	mkdir -p build
+	cp site/* build
+	cp src/index.css build
+	${BIN}/browserify site/index.js -o build/index.js -t [ babelify --presets [ es2015 ] ]
```

Comment:
Sorry, just got home and tested it without the presets, it doesn't seems to pickup the config of the `babelrc`, so we need those two I think.